websites/N/Necord/metadata.json

```diff
@@ -0,0 +1,54 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "someone",
+		"id": "235413185639874561"
+	},
+	"service": "Necord",
+	"description": {
+		"en": "A module for creating Discord bots using NestJS, based on Discord.js"
+	},
+	"url": "necord.org",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/QkxCnmF.png",
+	"thumbnail": "https://i.imgur.com/9xM4W3z.png",
+	"color": "#c12549",
+	"category": "other",
+	"tags": [
+		"open-source",
+		"documentation",
+		"discord"
+	],
+	"settings": [
+		{
+			"id": "search",
+			"title": "Show search query",
+			"icon": "fas fa-search",
+			"value": true
+		},
+		{
+			"id": "docs-page",
+			"title": "Show current page",
+			"icon": "fas fa-book",
+			"value": true
+		},
+		{
+			"id": "docs-page-content",
+			"title": "Show current content",
+			"icon": "fas fa-book",
+			"value": true
+		},
+		{
+			"id": "docs-page-sidebar",
+			"title": "Show selecting category",
+			"icon": "fas fa-book",
+			"value": true
+		},
```

Comment:
Why dont u just combine them into a privacy setting?

Reply:
Added from first example which I saw, so could you send link for documentation about setting's group?

Reply:
More flexibility?

Reply:
So you could add the privacy setting globally and put an if on each to untick them automatically when the privacy is ticked

Reply:
> So you could add the privacy setting globally and put an if on each to untick them automatically when the privacy is ticked

Agreed.

Reply:
Added privacy mode setting